### DISCUSSION TITLE
Newbie issues

### DISCUSSION THREAD

**Discussion by EVCV** [medusajs/medusa#715] [Q&A] [answered]
Hello everyone, 
I have previously dabbled in code etc and had a few websites that were okay but nothing special.
I started my own Shopify shop up but don't like not being in control of everything.

I came across Medusa today and have been looking into it but I am confused now as it seems over my head. I have looked at some tutorials but again seem over my head.

Is there anyone here that would be able to take a newbie under their wing and give some education to me? I feel like an idiot even asking this but I got lost and now I need to try to find the light to make things right (wow I can rhyme)

Thanks for your time
Richard

**Comment by srindom** [comment, marked as answer]
Hi @EVCV! We offer personal 1:1 onboardings to get you up and running :D 

[Please join our Discord](https://discord.gg/xpCwq3Kfn8) and ping me there then we can arrange a call!

**Reply by EVCV** [reply]
OMG that would be amazing, I am installing/signing up for Discord now and will Ping you when that is done. Thank you! I have potential for 4 or 5 shops I could set up eventually so I will be a happy chappy.
